release/models/isis/openconfig-isis-policy.yang

```diff
@@ -26,7 +26,13 @@ module openconfig-isis-policy {
     It augments the base routing-policy module with BGP-specific
     options for conditions and actions.";
 
-  oc-ext:openconfig-version "0.5.0";
+  oc-ext:openconfig-version "0.5.1";
```

Comment:
I think the minor version should be incremented since this is an API change:
http://103.17.196.104.bc.googleusercontent.com/docs/semver/

```
Semver guidelines should be followed while the model is at major number 0, i.e., API or feature changes should increment the minor number, while minor fixes should increment the patch number.
```

I'll create a PR to merge this doc into our release doc for Monday.